src/components/ProjectListComponent.jsx

```diff
@@ -1,5 +1,14 @@
 import React from "react";
+import {useStore} from "../store/store";
 // Things to display: A list of project titles from global store.
+
 export default function ProjectListComponent() {
-  return <div>profile list</div>;
```

Comment:
is this line still needed because you have 2 returns in this function

Reply:
The stuff in red with a minus is the stuff we took out.

Reply:
ok